server/api/lineitems.js

```diff
@@ -1,8 +1,30 @@
 const router = require('express').Router();
+const LineItem = require('../db/LineItem');
```

Comment:
Now that we're getting closer, I dont think we'll need api/lineitems.

https://acme-cart.herokuapp.com/ prof did the below: He added the route to the *product*. Which starts to make more sense. Perhaps we should do the same. 

app.post('/:id/lineItems', (req, res, next)=> {
  Order.addProductToCart(req.body.productId*1)
    .then( ()=> res.redirect('/'))
    .catch(next);
});

Reply:
I think Rav already implemented this in the orders.js.
We should keep it consistent. if addProductToCart is Order model method, then let's keep the api route in orders.js.

Reply:
Im okay with it if you guys are.